### PR TITLE
PropSWrapper interactive shell

### PR DESCRIPTION
**Pull request by gabrielStanovsky**
Adding an interactive shell for PropS wrapper. When run without parameters, it will display a shell, waiting for a raw sentence from the user, and then present the output PropS representation.

### CODE REVIEW

**Comment by OriShapira** [comment]
Looks good.

File: src/baseline_system/parsers/props_wrapper.py [hunk 1]
```diff
@@ -514,22 +515,40 @@ def _gensym_ent(self):
...
+            okrs.append(okr)
+            logging.info(pformat(okr))
+
+        # Dump json
+        with open(output_fn, 'w') as fout:
+            json.dump(okrs, fout)
+
+    else:
+        # Interactive shell
+        logging.info("Input a raw sentence. Press enter to finish each sentence.")
```
Comment (kleinay) [inline src/baseline_system/parsers/props_wrapper.py:542]: why logging.info and not simply "print"?

**Comment by gabrielStanovsky** [comment]
With logging you can control the level of verbosity you want and filter out prints.
I think it's good practice, but it doesn't really matter much here.

**Comment by gabrielStanovsky** [comment]
BTW, the previous version of the file also used logging.info in the example :)

File: src/baseline_system/parsers/props_wrapper.py (hunk 1 above)
Comment (gabrielStanovsky) [reply src/baseline_system/parsers/props_wrapper.py:542]: See my answer below.
Okay to merge, other than that?

Comment (kleinay) [reply src/baseline_system/parsers/props_wrapper.py:542]: Sure. merged.

**Pull request merged**